htmlRebuild.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Stance Nation</title>
+  </head>
+  <body>
+    <header>
+      <h1>OLD SCHOOL PERFECTION // TAKANOBU’S TOYOTA SPRINTER TRUENO.</h1>
+      <aside>Advertisement</aside>
+      <nav>
+        <ul>
+          <li>HOME</li>
+          <li>CAR FEATURES</li>
+          <li>EVENT COVERAGE</li>
+          <li>AROUND THE WEB</li>
+          <li>FORUM</li>
+          <li>ONLINE SHOP</li>
+        </ul>
+      </nav>
+      <section>
```

Comment:
There isn't really a reason to put this in a section. I would have just put the h1 inside of the article.